content/posts/2025-04-07_trunk_and_tidbits-march_2025/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Trunk & Tidbits, March 2025"
+description: "News from the Mastodon engineering team. Changes and work-in-progress from March 2025."
+date: 2025-04-07
+section: Trunk and Tidbits
+categories:
+- Trunk and Tidbits
+- Engineering
+tags:
+- engineering
+- mastodon
+authors:
+- renchap
+- andy
+resources:
+- name: hero
+  src: hero.png
+draft: true
+---
+
+Greetings! We're back, with our monthly update from the Mastodon engineering team.
+
+Remember how last month was a bit lighter for news? Well, this time, we are making up for it and have a lot to share! Take a look below for team updates, some significant new features, and updates to the mobile apps. We also have very important information about  changes with new (and old) software versions. Finally, there's a bumper round-up of community news, and some fun projects you may have missed.
+
+## Events and team news
+
+We are very excited to welcome [Echo](https://tech.lgbt/@chaosexanima) to the core team as a front-end developer. We extended our recruitment to include a second front-end developer, and found another great individual who will be joining us soon as well. Also on the team side, we opened [a role for a designer to join the team](https://mastodon.social/@MastodonEngineering/114235500948298378), after we said goodbye to Sam recently.
+
+In March, [Andy](https://macaw.social/@andypiper) was on stage at [Fediverse House](https://about.flipboard.com/fediverse-house/), a side event at SXSW run by our friends from Flipboard. This was a fun space which created opportunities to finally meet some of the members of our community in person (hi, [@box464](https://mastodon.social/@box464!)) and to talk about Mastodon and the Fediverse. Andy was also [a guest on the Fireside Fedi podcast](https://write.firesidefedi.live/firesidefedi/fireside-fedi-episode-7-andy-piper-mastodon), which you can watch as a video, or listen to in audio.
+
+If you have an account on one of our hosted instances (mastodon.social and mastodon.online), you may like to know that we now have an account that is run by the team that operates them - [@staff@mastodon.social](https://mastodon.social/@staff) is where you can find updates and announcements related to these services. As noted in the account bio, you should continue to use the existing process for any account support and appeals requests.
+
+Don't forget that we also post technical and team updates on [@MastodonEngineering@mastodon.social](https://mastodon.social/@MastodonEngineering).
+
+You're also free to check out our [LinkedIn page](https://www.linkedin.com/company/joinmastodon/) for non-engineering updates. It doesn't federate, but, we're there too.
+
+## Releases and updates
+
+In March, we released a number of bugfixes. You should take a look at the most recent version for the Mastodon level you are running, but also check the release notes for any versions you may also have missed. In total, we shipped:
```

Comment:
It's not only bugfixes, 4.3.6 contains security fixes

Reply:
thanks!